packaging/suse/portus.spec.in

```diff
@@ -18,6 +18,15 @@
 %define branch    __BRANCH__
 %define portusdir /srv/Portus
 
+%define fix_sheb() ( \
+  for i in $(grep '^#!/usr/bin/env ruby$' * -r | cut -d: -f1 | sort | uniq);do\
+    sed -e 's|^#!/usr/bin/env ruby$|#!/usr/bin/ruby.%{rb_suffix}|g' -i $i;\
+  done;\
+  for i in $(grep '^#!/usr/bin/ruby$' * -r | cut -d: -f1 | sort | uniq);do\
+     sed -e 's|^#!/usr/bin/ruby$|#!/usr/bin/ruby.ruby2.5|g' -i $i;\
+  done;\
+)
+
```

Comment:
I submitted a PR to the ruby-packaging with this macro:
 https://github.com/openSUSE/ruby-packaging/pull/9
Until we have it accepted, we need this macro definition here.